### PR TITLE
Developers should now develop locally instead of Vagrant

### PR DESCRIPTION
**Pull request by brandonchinn178**
@AdityaBhumbla can you have people running each OS (Windows, Mac, Linux/Ubuntu) and try these instructions? Let me know what's confusing for intro comp-comm people, if there's anything I should reword.

Also note how easy it is to get things set up. Everything should be fairly easy; I've installed `conda` on all 3 operating systems without much hassle, and Docker should be pretty cross-platform as well. But things may not go according to plan (Windows users might need to install Ruby/Sass?)

### CODE REVIEW

**Comment by brandonchinn178** [comment]
@AdityaBhumbla End-to-end tests now fail locally. I think it's the same issue as Circle CI, so hopefully when this is fixed, we can enable e2e tests on Circle CI again. Leaving to you

**Comment by rmpowell77** [comment]
question: Why use conda and brew for package management?  Why not just get python3.6 from brew?

**Comment by brandonchinn178** [comment]
I really like the way conda separates different Python versions, but it
could definitely be a "Mac use brew, otherwise use conda" step. I wanted to
have things as unified as possible across platforms, and `conda` is universal across OSes; installing node differs for each platform anyway, so might as well provide the `brew` command for that

File: README.md [hunk 1]
```diff
@@ -10,16 +10,59 @@ Goals:
...
+### Node.js
+
+Most of the code in this project is written using Vue, a Javascript framework for building client-side applications. In order to run and build this code, developers need to install Node from the [official site](https://nodejs.org/en/download/). For MacOS developers who use Homebrew, `brew install node` should suffice.
+
+After installing Node, run `npm install` to install the dependencies.
+
+### Django
+
+After finishing all the above installation steps, do the following:
+1. Install Django requirements (inside the virtual environment): `pip install -r requirements/dev.txt`
```
Comment (rmpowell77) [inline README.md:59]: I found I had to run ```pip3 install -r requirements/dev.txt```

**Comment by brandonchinn178** [comment]
After running `brew install python3.6`? This command should work for conda.

**Comment by brandonchinn178** [comment]
Just gonna merge and deal with problems later

**Pull request merged**